Library/Homebrew/formula_assertions.rb

```diff
@@ -8,8 +8,31 @@ module Homebrew
   module Assertions
     include Context
 
-    require "test/unit/assertions"
-    include ::Test::Unit::Assertions
+    require "minitest"
+    require "minitest/assertions"
+    include ::Minitest::Assertions
+
+    attr_writer :assertions
+
+    def assertions
+      @assertions ||= 0
+    end
+
+    # Test::Unit backwards compatibility methods
```

Comment:
Deprecate these (or at least a commented out deprecation for now) maybe?

Reply:
Could do, yes.

Bonus (not that it matters much): all the "new" methods actually already exist in `Test::Unit` too, as aliases.